buildchain/buildchain/codegen.py

```diff
@@ -14,6 +14,21 @@
 from buildchain import utils
 
 
+def get_task_information() -> types.TaskDict:
+    """Retrieve all the task information from codegen"""
+    result = {
+        "actions": [],
+        "task_dep": [],
+        "file_dep": [],
+    }  # type: types.TaskDict
```

Comment:
To stay consistent with other type annotations in the `buildchain` package:

```suggestion
    result: types.TaskDict = {
        "actions": [],
        "task_dep": [],
        "file_dep": [],
    }
```

This also should remove the need for having "TaskDict" in our `.pylint-dict`

Reply:
right